chef/cookbooks/cpe_preferencepanes/resources/default.rb

```diff
@@ -15,14 +15,13 @@
 resource_name :cpe_preferencepanes
 default_action :config
 
+# rubocop:disable Metrics/BlockLength
 action :config do
   prefs = node['cpe_preferencepanes'].reject { |_k, v| v.nil? }
```

Comment:
As in the rubocop disable line? Yep! This was causing it to fail and it no longer does.